vmdb/db/migrate/20150604092700_add_token_to_authentications.rb

```diff
@@ -0,0 +1,9 @@
+class AddTokenToAuthentications < ActiveRecord::Migration
+  def up
+    add_column :authentications, :token, :text
```

Comment:
@Fryguy @blomquisg do you have something similar (or a similar need) in openstack?